packages/sdk/package.json

```diff
@@ -26,12 +28,17 @@
     "js"
   ],
   "scripts": {
-    "build": "yarn tsc",
+    "build": "tsup",
     "lint": "eslint",
     "lint:fix": "yarn lint --fix",
+    "format": "prettier src/ --check",
+    "format:fix": "prettier src/ --write",
```

Comment:
This command can be replaced with `lint`.

Reply:
What do you mean by that ?
If we only have `lint` it means we would have scripts like:
```json
"lint": "eslint --fix && prettier src/ --check",
"lint:fix": "eslint --fix && prettier src/ --write" 
```
I find it clearer like currently but why not.
And if you meant including prettier as an ESLint plugin so we only have to run ESLint, I don't like doing this for 2 reasons:
- While you code it bloats your IDE with warnings/errors about the format (if you don't follow the rules of course) until you save. Imo a formatting tool should just run silently on save to keep consistency in the codebase. If someone isn't accustomed with the styling rules of a project, he shouldn't be bothered with warnings about it, it should just be fixed automatically for him. 
- It's slower to run

This article sums it up pretty well: https://www.joshuakgoldberg.com/blog/you-probably-dont-need-eslint-config-prettier-or-eslint-plugin-prettier/#eslint-plugin-prettier (the author is also against `eslint-config-prettier`, but that's another discussion and a less important one)

Reply:
Thanks for sharing your thoughts 🙏 
I understand your intentions.

Adding prettier's commands to explicitly separate them seems like a good idea.

However, my personal opinion is that regardless of performance, we should inspect our code with lint tools and formatting tools when we save it.

Because in the end, the code we need to manage is the code that has been checked with lint tools and formatting tools.

I also think that eslint's prettier plugin is a powerful feature that can provide a unified management point.
As the project grows in size, performance issues can arise as we feared, but the lint cache and prettier cache options can improve performance.

Additionally, the script will work fine if prettier is run first.
```json
"lint": "prettier src/ --check && eslint",
"lint:fix": "prettier src/ --write && eslint --fix" 
```

Reply:
> However, IMHO that regardless of performance, we should inspect our code with lint tools and formatting tools when we save it.

> Because in the end, the code we need to manage is the code that has been checked with lint tools and formatting tools.

I completely agree on this yes, my point is more on the unified run with the plugin. Performance is a secondary problem it shouldn't be really annoying, the main "pain point" of this imo is having warnings thrown by ESLint due to formatting, but that will disappear by themselves when hitting CTRL S. 

Anyway it's not a very very important topic ahah, let's unify the script as you suggested!